packages/gestalt/src/RadioGroupButton.js

```diff
@@ -0,0 +1,192 @@
+// @flow strict
+import { forwardRef, type Node, useState } from 'react';
+import classnames from 'classnames';
+import controlStyles from './RadioButtonCheckbox.css';
+import styles from './RadioButton.css';
+import Box from './Box.js';
+import Label from './Label.js';
+import Text from './Text.js';
+import { type AbstractEventHandler } from './AbstractEventHandler.js';
+import useFocusVisible from './useFocusVisible.js';
+import focusStyles from './Focus.css';
+import { useRadioGroupContext } from './RadioGroupContext.js';
+
+type Props = {|
+  /**
+   * Indicates if the input is checked. See the [state example](https://gestalt.pinterest.systems/radiogroup#States) for more details.
+   */
+  checked?: boolean,
+  /**
+   * Indicates if the input is disabled. See the [state example](https://gestalt.pinterest.systems/radiogroup#States) for more details.
+   */
+  disabled?: boolean,
+  /**
+   * A unique identifier for the input.
+   */
+  id: string,
+  /**
+   * An optional [Image](https://gestalt.pinterest.systems/image) component can be supplied to add an image to each radio button. Spacing is already accounted for — simply specify the width and height. See the [images example](https://gestalt.pinterest.systems/radiogroup#With-Image) for more details.
+   */
+  image?: Node,
+  /**
+   * The displayed label for the input.
+   */
+  label?: string,
+  /**
+   * The name given for all radio buttons in a single group.
+   */
+  name?: string,
+  /**
+   * Callback triggered when the user interacts with the input.
+   */
+  onChange: AbstractEventHandler<SyntheticInputEvent<HTMLInputElement>, {| checked: boolean |}>,
```

Comment:
Since we don't define AbstractEventHandler anywhere in the docs, we've been inlining that type definition where it's used

Reply:
is there an example of that?

Reply:
@ayeshakmaz  This one is 

`onChange: ({| event: SyntheticInputEvent<HTMLInputElement>, checked: boolean |}) => void,
`

Reply:
Ahhh I see, thank you!